app/code/community/Nexcessnet/Turpentine/Model/Observer/Esi.php

```diff
@@ -528,45 +528,29 @@ protected function _checkIsEsiUrl($url) {
         return ! $this->_checkIsNotEsiUrl($url);
     }
 
-    public function hookToControllerActionPreDispatch($observer) {
-        if (Mage::helper('turpentine/data')->getVclFix() == 0 && $observer->getEvent()->getControllerAction()->getFullActionName() == 'checkout_cart_add') {
-            Mage::dispatchEvent("add_to_cart_before", array('request' => $observer->getControllerAction()->getRequest()));
-        }
-        if ($observer->getEvent()->getControllerAction()->getFullActionName() == 'wishlist_index_index') {
-            Mage::dispatchEvent('wishlist_index_index_before', array('request' => $observer->getControllerAction()->getRequest()));
-        }
-    }
-
-    public function hookToControllerActionPostDispatch($observer) {
```

Comment:
There was some code commented out in the xml so I removed that and the methods that it was referencing. That is, this and `hookToAddToCartAfter`.